src/Panel.tsx

```
@@ -200,6 +205,9 @@ export default function Panel<
             toggleOpen={noop}
             open
             direction={direction}
+            defaultActiveKey={defaultActiveKey}
```

Biome

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)